### PR TITLE
add info on GCP label character limitations

### PR DESCRIPTION
**Pull request by chrisahl**
For https://issues.redhat.com/browse/AAP-14176 and release-2.4

### CODE REVIEW

**Review by leena-jawale — APPROVED** [review]

File: stories/topics/proc-gcp-deploy-with-existing-vpc.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ Select *cloud NAT router* to create a *NAT router* in your VPC network.
 . The *Networking section* provides defaults for all of the network ranges used in the deployment. 
 Provide these values based on your existing network configuration. 
 If you want to modify these values, see xref:ref-gcp-networking-options[Networking Options] 
-. Optional: In the *Additional Labels* section, provide any additional label key and value pairs to be added to the GCP resources that are part of the deployment.
+. Optional: In the *Additional Labels* section, provide any additional label key and value pairs to be added to the GCP resources that are part of the deployment. Valid keys and values must meet GCP label requirements.  For a *key*, only hyphens, underscores, lowercase characters and numbers are allowed. A *key* must start with a lowercase character. For a *value*, only hyphens, underscores, lowercase characters and numbers are allowed.
```
Comment (ianf77) [inline stories/topics/proc-gcp-deploy-with-existing-vpc.adoc:27]: are permitted.

File: stories/topics/proc-gcp-deploy-with-new-vpc.adoc [hunk 2]
```diff
@@ -16,7 +16,7 @@ The zone must be in the Region you selected.
...
 . In the *Network Selection* section, select *New network*.
 The Networking section provides defaults for all of the network ranges used in the deployment. If you want to modify these values, see xref:ref-gcp-networking-options[Networking Options].
-. Optional: In the *Additional Labels* section, provide any additional label key and value pairs to be added to the GCP resources that are part of the deployment.
+. Optional: In the *Additional Labels* section, provide any additional label key and value pairs to be added to the GCP resources that are part of the deployment. Valid keys and values must meet GCP label requirements.  For a *key*, only hyphens, underscores, lowercase characters and numbers are allowed. A *key* must start with a lowercase character. For a *value*, only hyphens, underscores, lowercase characters and numbers are allowed.
```
Comment (ianf77) [inline stories/topics/proc-gcp-deploy-with-new-vpc.adoc:19]: are permitted.

**Review by ianf77 — COMMENTED** [review]
Couple of minor comments.

File: stories/topics/proc-gcp-deploy-with-existing-vpc.adoc (hunk 1 above)
Comment (chrisahl) [reply stories/topics/proc-gcp-deploy-with-existing-vpc.adoc]: fixed

File: stories/topics/proc-gcp-deploy-with-new-vpc.adoc (hunk 2 above)
Comment (chrisahl) [reply stories/topics/proc-gcp-deploy-with-new-vpc.adoc]: fixed

**Pull request merged**